src/data/requestFiltering.js

```diff
@@ -82,6 +98,17 @@ export function populateRequestFilterFrom(queryStringData, me, events) {
   }
 }
 
+function matchByRegion (location, requestLocation) {
+  if (!location || !requestLocation) return true
+  if (location.sublocality || location.locality || location.county) {
+    return isNear(location, requestLocation)
+  } else if (location.state) {
+    return requestLocation.description.includes(location.state)  || requestLocation.description.includes(location.description)
+  } else if (location.country) {
+    return requestLocation.description.includes(location.country) || requestLocation.description.includes(location.description)
```

Comment:
Same as with state. It may be included, but not spelled or capitalized the same way.